### ISSUE TITLE
Higher Versions of React-Native

### ISSUE THREAD

**Issue by JoeVelliquette** [chaimPaneth/react-native-jw-media-player#6]
Does this package work with react-native >= 0.60?

**Comment by KestasVenslauskas** [comment]
Having trouble running this on react-native >= 0.60....
@JoeVelliquette Did you manage to make it work?

**Comment by chaimPaneth** [comment]
Hi @JoeVelliquette & @KestasVenslauskas 
I've updated the library and the latest versions are compatible with react-native >= 0.60

**Comment by howlCode** [comment]
Are the instructions for installation still the same? Is the package now automatically linked? I cannot get this to work.

**Comment by chaimPaneth** [comment]
@howlCode  it is automatically linked I will update the readme soon enough. 
What is the error you're getting?

**Comment by howlCode** [comment]
@chaimPaneth Awesome, thanks for the update! Unfortunately, it still isn't working for me but I did finally get the package to mount in the view. I had to take the following steps FYI:
in android/build.gradle I had to add the following to repositories:
`        maven {
            url 'http://mvn.jwplayer.com/content/repositories/releases/'
        }
`

...and in android/app/build.gradle I had to add the following to dependencies:

`
    api 'com.google.android.gms:play-services-base:+'
`

But now I'm getting a jwPlayer error code 100012, which references a bad API key, though I do have it set in the AndroidManifest as instructed.
